azure-pipelines.yaml

```diff
@@ -1,102 +1,103 @@
 name: $(Build.SourceBranchName)_$(Build.Reason)_$(majorVersion).$(Build.BuildId)$(Rev:.r)
 variables:
-  buildConfiguration: 'Release'
+  buildConfiguration: Release
   majorVersion: 0.0.0
 
 # Enable PR validation on branches master and dev
 pr:
   branches:
     include:
-    - master
-    - dev
+      - master
+      - dev
 
 # Enable CI on branches master and dev
 # Batch builds
 trigger:
   batch: true
   branches:
     include:
-    - dev
-    - master
+      - dev
+      - master
 
 jobs:
+  - job: build_and_test
+    displayName: Build and Test
+    continueOnError: false
+    pool:
+      name: 1ES-Hosted-Azfunc
+      demands:
+        - ImageOverride -equals MMSUbuntu20.04TLS
```

Comment:
Only line 29 is real change. Rest of the changes are result of correcting spellings and running auto-formatter on the document.